cmd/earth/main.go

```diff
@@ -319,14 +345,32 @@ func (app *earthApp) parseConfigFile(context *cli.Context) error {
 		}
 	}
 
-	gitConfig, gitCredentials, err := config.CreateGitConfig(cfg)
-	if err != nil {
-		return errors.Wrapf(err, "failed to create git config from %s", app.configPath)
+	if context.IsSet("git-url-instead-of") {
+		app.console.Warnf("Warning: the --git-url-instead-of command flag is deprecated and is now configured in the earthly config file under the `git` global url_instead_of setting\n")
+	} else {
+		if gitGlobal, ok := cfg.Git["global"]; ok {
+			if gitGlobal.GitURLInsteadOf != "" {
+				app.buildkitdSettings.GitURLInsteadOf = gitGlobal.GitURLInsteadOf
+			}
+		}
+	}
+
+	if context.IsSet("no-loop-device") {
+		app.console.Warnf("Warning: the --no-loop-device command flag is deprecated and is now configured in the earthly config file under the `no_loop_device` setting\n")
```

Comment:
Should warnings only have the prefix colored (eg `Warning`) and leave rest of the text as default color? I'm thinking making the whole line red might be too much?

Reply:
I was unsure about that too, if it's just the "warning", then the eye is drawn to that more so than equally drawn to the entire line. 

Here's what it currently looks like:

![image](https://user-images.githubusercontent.com/1806823/88101479-e5b7b480-cb52-11ea-831f-e2643e03fb6a.png)

Reply:
Then compare with only the prefix:

![image](https://user-images.githubusercontent.com/1806823/88103972-9e332780-cb56-11ea-9e52-a4fc17669c66.png)

maybe it's fine just as-is with the prefix only highlighted.

Reply:
Ok sounds good